apps/playground/src/app/components/forms/input-box/input-box-visual-host.component.ts

```diff
@@ -24,6 +24,7 @@ export class InputBoxVisualHostComponent implements OnInit {
     if (this.inputBoxHostSvc) {
       this.inputBoxHostSvc.populate({
         inputTemplate: this.inputTemplateRef,
+        hintText: 'Host hint text.',
```

Comment:
This throws an error.

Reply:
Fixed - I thought I got all of these - but apparently missed the playground.